pkg.sh

```diff
@@ -33,25 +41,12 @@ GOOS=darwin GOARCH=amd64 go build -trimpath -o ../../../bin/dexc-darwin-amd64-v$
 GOOS=darwin GOARCH=arm64 go build -trimpath -o ../../../bin/dexc-darwin-arm64-v${VER} -ldflags "$LDFLAGS"
 popd
 
-pushd client/webserver/site
-go generate # should be a no-op
-npm ci
-npm run build
-popd
-
-rm -rf bin/site
-mkdir -p bin/site/src
-pushd client/webserver/site
-cp -R dist ../../../bin/site
-cp -R src/font src/html src/localized_html src/img ../../../bin/site/src
-popd
+echo "Files embedded in the Go webserver package:"
+go list -f '{{ .EmbedFiles }}' decred.org/dcrdex/client/webserver
```

Comment:
Here's how we'll verify that the binaries have the intended site files embedded.

Reply:
I think I'm mistaken about the go list command.  e.g. I make the build with the dist dir and it embeds, then I can do the go list command and it shows the files.  Then I `rm -r dist` and do the list command again:

```
$  go list -f '{{ .EmbedFiles }}' decred.org/dcrdex/client/webserver
../webserver.go:81:13: pattern site/dist: no matching files found
```

So clearly that go list is not just checking the compiled webserver package in the go package cache or whatever, but something based on the current files.

In this pkg script it probably is doing what we expect, but in general it's not.